finance/funding_requests.md

```diff
@@ -0,0 +1,52 @@
+
+Funding requests can be made to the Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 
+
+
+## Travel
+
+1. Send an email to astropy-finance-committee@googlegroups.com with the following information:
+* Name
+* Description
+* Total Request
+* Itemized Budget
+
+Costs for per diem can use the [US Government per deim rates | https://www.gsa.gov/travel/plan-book/per-diem-rates], and airfare costs should be justified. 
+
+2. The financd committee will review the costs and provide approval within 1 week of the request. 
```

Comment:
interim finance committee

Reply:
will aim to provide approval within 1 week of the request.

Reply:
Ideally, at least 1 month before travel is to commence. Include relevance to the Project-wide goals.

Reply:
Refer to the travel guidelines issue....which is still in the private repo. It was on my list to move them...boo....